app/views/register/confirmation/newTrust/AgentView.scala.html

```diff
@@ -33,27 +33,37 @@
 
     <p class="panel-indent">@components.link(ConfirmationAnswerPageController.onPageLoad(draftId).url, "print-and-save", "confirmation.print.link", openInNewWindow = false)</p>
 
-    <h2>@messages("confirmation.new.agent.subheading1")</h2>
-
-    <p>@messages("confirmation.new.agent.p1", leadTrusteeName)</p>
-
-    <h2>@messages("confirmation.new.agent.subheading2")</h2>
-
-    <p>@messages("confirmation.new.agent.p2")</p>
-    <p>@messages("confirmation.new.agent.p3")</p>
-
-    <ol class="govuk-list--number">
-        <li class="bold">@messages("confirmation.new.agent.bullet1")
-            <p>@messages("confirmation.new.agent.bullet1.p1")</p>
-            <p>@messages("confirmation.new.agent.bullet1.p2") <a href="@config.ggSignInUrl" target="_blank">@messages("confirmation.new.agent.bullet1.p2.a")</a>.</p>
-            <p>@messages("confirmation.new.agent.bullet1.p3")</p>
-            <p>@messages("confirmation.new.agent.bullet1.p4")</p>
-        </li>
-        <li class="bold">@messages("confirmation.new.agent.bullet2")
-            <p>@messages("confirmation.new.agent.bullet2.p1")</p>
-            <p>@messages("confirmation.new.agent.bullet2.p2")</p>
-            <p>@messages("confirmation.new.agent.bullet2.p3") <a href="@config.helplineUrl" target="_blank">@messages("confirmation.new.agent.bullet2.p3.a")</a>.</p>
-            <p>@messages("confirmation.new.agent.bullet2.p4") @components.link(controllers.register.agents.routes.AgentOverviewController.onPageLoad().url, "agent-overview", "confirmation.new.agent.bullet2.p4.a", false).</p>
-        </li>
-    </ol>
+    <p class="panel-indent">@messages("confirmation.declare.warning")</p>
+
+    <h2>@messages("confirmation.nonTaxable.agent.subheading1")</h2>
+        <p>@messages("confirmation.nonTaxable.agent.p1", leadTrusteeName)</p>
+        <p>@messages("confirmation.nonTaxable.agent.p2")</p>
+        <p>@messages("confirmation.nonTaxable.agent.p3") <a href="@config.helplineUrl" target="_blank">@messages("confirmation.nonTaxable.agent.link1")</a>.</p>
+
+    <h2>@messages("confirmation.nonTaxable.agent.subheading2")</h2>
+        <p>@messages("confirmation.nonTaxable.agent.p4")</p>
+        <p>@messages("confirmation.nonTaxable.agent.p5", leadTrusteeName)</p>
+            <li class="bold">
+                <p>@messages("confirmation.nonTaxable.agent.b1")</p>
+                <p>@messages("confirmation.nonTaxable.agent.b2")</p>
+            </li>
+        <p>@messages("confirmation.nonTaxable.agent.p6")</p>
+        <p>@messages("confirmation.nonTaxable.agent.p7")</p>
+
+    <h2>@messages("confirmation.nonTaxable.agent.subheading3")</h2>
+        <p>@messages("confirmation.nonTaxable.agent.p8", leadTrusteeName)</p>
+        <p>@messages("confirmation.nonTaxable.agent.p9")</p>
+        <p>@messages("confirmation.nonTaxable.agent.p10") <a href="config.ggSignInUrl" target="_blank">@messages("confirmation.nonTaxable.agent.link2")</a></p>
```

Comment:
This `config.ggSignInUrl` will not render the link correct, have you tested this?

Reply:
It needs the `@` at the start to interpret the scala code.